File: backend/ProjectProgressManagementSystem/Models/Project.cs
namespace ProjectProgressManagementSystem.Models
{
    public class Project : EntityBase
    {
        public string Number { get; set; }
        public string Title { get; set; }
        public string? Business { get; set; }
        public Department Department { get; set; }
        public Region Region { get; set; }
        public string? Description { get; set; }
        public List<WeekData> WeekData { get; set; } = new List<WeekData>();
    }
}

File: backend/app/models/project.py
from sqlalchemy import Integer, String
from sqlalchemy.orm import Mapped, mapped_column, relationship

from app.models.base_entity import EntityBase


class Project(EntityBase):
    __tablename__ = "Project"

    number: Mapped[str] = mapped_column(String(100), nullable=False)
    title: Mapped[str] = mapped_column(String(255), nullable=False)
    business: Mapped[str | None] = mapped_column(String(255), nullable=True)
    department: Mapped[int] = mapped_column(Integer, nullable=False)
    region: Mapped[int] = mapped_column(Integer, nullable=False)
    description: Mapped[str | None] = mapped_column(String(1000), nullable=True)

    week_data: Mapped[list["WeekData"]] = relationship(back_populates="project")  # noqa: F821
